src/ui/Windows/MainFile.cpp

```diff
@@ -3778,10 +3778,16 @@ LRESULT DboxMain::SynchCompareResult(PWScore *pfromcore, PWScore *ptocore,
              sxSync_DateTime, IDSC_SYNCPOLICY);
         if (pPolicyCmd != NULL)
           pmulticmds->Add(pPolicyCmd);
+      } else if (i == 8) { // XXX hack test theory
```

Comment:
Did you mean to keep this XXX comment? If so, it should have more context to help future maintainers.

Also, is 8 one of the values of `FieldType` - could we use that instead?

Reply:
Oops - wrong commit pushed...
More work due in any case - thanks!